firmware/quadruna/VC/src/hw/hw_tasks.h

```diff
@@ -0,0 +1,27 @@
+#include "io_can.h"
+#include "hw_can.h"
+#include "configs/App_HeartbeatMonitorConfig.h"
+#include "App_SharedStateMachine.h"
+#include "App_SharedHeartbeatMonitor.h"
+
+extern struct StateMachine *    state_machine;
+extern struct HeartbeatMonitor *heartbeat_monitor;
+
+typedef struct
+{
+    ADC_HandleTypeDef *      hadc1;
+    ADC_HandleTypeDef *      hadc3;
+    FDCAN_HandleTypeDef *    hfdcan1;
+    IWDG_HandleTypeDef *     hiwdg1;
+    const CanConfig *        can_config;
+    struct StateMachine *    state_machine;
+    struct HeartbeatMonitor *heartbeat_monitor;
+} HwTasksConfig;
```

Comment:
It might be easier to just `extern` these in the `hw_tasks.c` file. The reason we inject the dependencies into shared modules is so they can be reused across different boards, but the tasks file is always going to be specific to a single board. So just using `extern` to get them from the `main.c` file will probably be less boilerplate.